frontend/workflows/k8s/src/update-probe.tsx

```diff
@@ -0,0 +1,385 @@
+        }
+      }
+    });
+    updateRows.push({ name: "Probe type", value: probeType });
+  });
+
+  return (
+    <WizardStep error={updateData.error} isLoading={updateData.isLoading}>
+      <Confirmation action="Update" />
+      <MetadataTable
+        data={[
+          { name: "Name", value: deployment.name },
+          { name: "Namespace", value: deployment.namespace },
+          { name: "Cluster", value: deployment.cluster },
+          ...updateRows,
+        ]}
+      />
+    </WizardStep>
+  );
+};
+
+const UpdateLiveness: React.FC<WorkflowProps> = ({ heading, resolverType }) => {
+  interface BodyRequest {
+    clientset: any;
+    cluster: string;
+    namespace: string;
+    name: string;
+    fields: {
+      containerProbes: {
+        containerName: string;
+        [key: string]: any; // Permitir otros atributos dinámicos
+      }[];
+    };
+  }
+
+  const dataLayout = {
+    inputData: {},
+    deploymentData: {},
+    currentDeploymentData: {},
+    updateData: {
+      deps: ["deploymentData", "inputData", "currentDeploymentData"],
+      hydrator: (
+        deploymentData: {
+          cluster: string;
+          containerName: string;
+          deploymentSpec: IClutch.k8s.v1.Deployment.DeploymentSpec;
+          name: string;
+          namespace: string;
+        },
+        inputData: { clientset: string },
+        currentDeploymentData: IClutch.k8s.v1.Deployment
+      ) => {
+        const clientset = inputData.clientset ?? "undefined";
+        const container = findContainer({ ...deploymentData });
+        const bodyRequest: BodyRequest = {
+          clientset,
+          cluster: deploymentData.cluster,
+          namespace: deploymentData.namespace,
+          name: deploymentData.name,
+          fields: {
+            containerProbes: [
+              {
+                containerName: deploymentData.containerName,
+              },
+            ],
+          },
+        };
+        if (Object.keys(container.livenessProbe).length !== 0) {
+          bodyRequest.fields.containerProbes[0].livenessProbe = {
+            timeoutSeconds: container.livenessProbe.timeoutSeconds,
+            initialDelaySeconds: container.livenessProbe.initialDelaySeconds,
+            periodSeconds: container.livenessProbe.periodSeconds,
+            successThreshold: container.livenessProbe.successThreshold,
+            failureThreshold: container.livenessProbe.failureThreshold,
+          };
+        }
+        if (Object.keys(container.readinessProbe).length !== 0) {
+          bodyRequest.fields.containerProbes[0].readinessProbe = {
+            timeoutSeconds: container.readinessProbe.timeoutSeconds,
+            initialDelaySeconds: container.readinessProbe.initialDelaySeconds,
+            periodSeconds: container.readinessProbe.periodSeconds,
+            successThreshold: container.readinessProbe.successThreshold,
+            failureThreshold: container.readinessProbe.failureThreshold,
+          };
+        }
```

Comment:
Should be able to clean these up a bit as well:

```suggestion
        ["livenessProbe", "readinessProbe"].forEach(probeType => {
          if (Object.keys(container[probeType]).length !== 0) {
            const {
              timeoutSeconds,
              initialDelaySeconds,
              periodSeconds,
              successThreshold,
              failureThreshold,
            } = container[probeType];

            bodyRequest.fields.containerProbes[0][probeType] = {
              timeoutSeconds,
              initialDelaySeconds,
              periodSeconds,
              successThreshold,
              failureThreshold,
            };
          }
        });
```

Reply:
Update attributes validations